drivers/telescope/lx200ap_experimental.cpp

```diff
@@ -198,16 +209,32 @@ bool LX200AstroPhysicsExperimental::updateProperties()
 
     if (isConnected())
     {
+        deleteProperty("TELESCOPE_PIER_SIDE");  
         defineText(&VersionInfo);
 
         /* Motion group */
         defineSwitch(&APSlewSpeedSP);
         defineSwitch(&SwapSP);
+#ifdef no
         defineSwitch(&SyncCMRSP);
+#endif
         defineSwitch(&APGuideSpeedSP);
         defineSwitch(&ParkToSP);
-        defineNumber(&MeridianDelayNP);
+        defineNumber(&APSiderealTimeNP);
+        defineNumber(&HourangleCoordsNP);
+        defineNumber(&APUTCOffsetNP);
 
+        IUResetSwitch(&ParkSP);
+        ParkS[0].s = ISS_ON ;
+        ParkSP.s = IPS_OK;
+        IDSetSwitch(&ParkSP, nullptr);
+
+
+#ifdef no
+        // 2020-05-30, wildi, this is the wrong place to load config data,
+        // see celestron driver.
+        // The RA/Dec values are 0,90 at startup and should not be
+        // displayed until they are set in UnPark()
         // load in config value for park to and initialize park position
         loadConfig(true, ParkToSP.name);
```

Comment:
I don't think loadConfig is the correct way here. You have functions to read each value separately now from the config file without having issue an ISNewSwitch(...) function call which is not required at this stage.

IUReadConfigSwitch should be used.

Reply:
I came to the same conclusion and I disabled this section from line 233 to 283 using #ifdef...#endif.